.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       RUBYOPT: '-rostruct'
     steps:
       - checkout
-      - run: gem install bundler -v '1.17'
+      - run: gem install bundler -v '2.5.23'
```

Comment:
Bumping this in the Ruby 3.1+ commit because we seemed to be getting failures without it.